soc/xtensa/intel_adsp/tools/cavstwist.sh

```diff
@@ -112,14 +112,14 @@ if [ "$DO_LOAD" = "1" ]; then
     # Must "accept" the file so the next signing script knows it's OK
     # to write a new one
     mv $IMAGE $IMAGE2
-    scp $IMAGE2 $CAVSTOOL $CAVS_HOST:
+    scp $IMAGE2 $CAVSTOOL scp://$CAVS_HOST
     rm -f $IMAGE2
 
     # Twister seems to overlap tests by a tiny bit, and of course the
     # remote system might have other junk running from manual testing.
     # If something is doing log reading at the moment of firmware
     # load, it tends fairly reliably to hang the DSP.  Kill it.
-    ssh $CAVS_HOST "sudo pkill -9 -f cavstool" ||:
+    ssh ssh://$CAVS_HOST "sudo pkill -9 -f cavstool" ||:
```

Comment:
I find this slightly more readable but it's off-topic
```suggestion
    ssh ssh://$CAVS_HOST "sudo pkill -9 -f cavstool" || true
```